### PR TITLE
Migration of ansible to collections -1

### PR DESCRIPTION
**Pull request by SHANDCRUZ**
### Description
ADDED RESOURCE
- oneview_alert_facts

### Issues Resolved
None

### Check List
- [ ] New functionality includes sanity testing.
  - [ ] All sanity tests pass. (`$ ansible-test sanity`).
  - [ ] All unit tests pass.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

**Review by VenkateshRavula — COMMENTED** [review]
Add CHANGELOG and endpoints-support.md files

**Review by VenkateshRavula — APPROVED** [review]

File: plugins/modules/oneview_alert_facts.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+extends_documentation_fragment:
+    - hpe.oneview.oneview
+    - hpe.oneview.oneview.factsparams
+    - hpe.oneview.oneview.params
+'''
+
+EXAMPLES = '''
+- name: Gather facts about the last 2 alerts
+  oneview_alert_facts:
+    hostname: 172.16.101.48
```
Comment (AsisBagga) [inline plugins/modules/oneview_alert_facts.py:54]: can you remove this IPs please, check for below examples too

File: plugins/modules/oneview_interconnect_link_topology_facts.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
```
Comment (AsisBagga) [inline plugins/modules/oneview_interconnect_link_topology_facts.py:4]: please update this

File: roles/oneview_alert_facts/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_alert_facts
+  author: Andressa Cruz
```
Comment (AsisBagga) [inline roles/oneview_alert_facts/meta/main.yml:4]: please update your name over here.

File: roles/oneview_interconnect_link_topology_facts/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_interconnect_link_topology_facts
+  author: Mariana Kreisig
```
Comment (AsisBagga) [inline roles/oneview_interconnect_link_topology_facts/meta/main.yml:4]: please update your name over here.

**Pull request closed without merging**